tools/build_go_exe.sh

```diff
@@ -28,6 +28,10 @@ fi
 GO_VERSION=$(go version | awk '{print $3}')
 # Note, UPX only works on binaries produced by Go 1.7+. However, we require Go 1.8+
 UPX_BINARY="$(which upx || which upx-ucl || echo '')"
+# For dev iteration; upx takes a long time; can set env var
+if [ -n "$CLI_BUILD_SKIP_UPX" ]; then
+    UPX_BINARY=
```

Comment:
The line above sets the value to '' when the binary isn't found, so this is consistent.